docs/en/connector-v2/source/Http.md

```diff
@@ -151,3 +164,7 @@ Http {
 ### 2.2.0-beta 2022-09-26
 
 - Add Http Source Connector
+
+### new version
+
+- Increase jsonpath parsing.
```

Comment:
Add PR link.

Reply:
Please refer to the format of other changed logs. `[XXX][XXX]`